<launch>
  <node name="collision_detection" pkg="ros_collision_detection" type="collision_detection" output="screen"/>
  <param name="ttc_algorithm_classname" value="NCircleAlgorithm" type="str" />
  <param name="ttc_algorithm_circle_count" value="2" type="int" />
</launch>